### PR TITLE
Feature/Support page

### PR DESCRIPTION
**Pull request by juanbravozu**
## Changes

- Added page file for Support us page.
- Added Testimony and Testimonies components with their respective styles.
- Added variation to Big image layout to support the layout with card proposed in Figma.
- Changed validation in Big image layout to check if there's a cta.

screencapture-localhost-3000-apoyanos-2021-09-28-19_14_09

### CODE REVIEW

File: pages/apoyanos.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          url: howTo.sectionImage.fields.file.url,
+          imgTitle: howTo.sectionImage.fields.title,
+          ...howTo,
+        }}
+      />
+      <Collaborate
+        fields={collaborate}
+      />
+      <TestimoniesWrapper
+        fields={testimonies}
```
Comment (tatarodriguez11) [inline pages/apoyanos.js:45]: testimonials*

**Review by cricar6 — APPROVED** [review]

File: components/Collaborate.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import BigImageLayout from './BigImageLayout';
```
Comment (Cruzlex7) [inline components/Collaborate.js:1]: Why not use BigImageLayout Directly? 
Is it in this Component where we should put the 'Formas de Colaborar' section? Where it should be?

**Comment by Cruzlex7** [comment]
Why we have that LSEP code in titles from the attached image?

File: components/Collaborate.js (hunk 2 above)
Comment (juanbravozu) [reply components/Collaborate.js:1]: I thought it'd be more organized if the data is deconstructed outside of the page file itself

**Comment by juanbravozu** [comment]
> Why do we have that LSEP code in titles from the attached image?

We're getting that character from contentful in multiple pages. This is an example from the hero: 

image

**Review by javiermlovo — APPROVED** [review]

**Review by tatarodriguez11 — APPROVED** [review]

**Review by javiermlovo — APPROVED** [review]

**Review by Cruzlex7 — APPROVED** [review]

**Review by cricar6 — APPROVED** [review]

**Pull request merged**